### DISCUSSION TITLE
First nbminer attempt... How to gauge success :)

### DISCUSSION THREAD

**Discussion by wxprofessor** [NebuTech/NBMiner#792] [Show and tell]
How does one gauge success with nbminer... In the 'old days' I would run Nicehash.. This is my new system running a 3080ti with custom water cooled setup, and my first attempt at trying nbminer :) ... here is the output so far, from the args:
`@cd /d "%~dp0"
nbminer -a ethash -o stratum+tcp://asia2.ethermine.org:14444 -u [wallet].default -d 0 --platform 1 --temperature-limit 72 -log
pause`

![image](https://user-images.githubusercontent.com/5704029/161440459-8e60ff95-4563-4c5d-83b4-f254ebe7b8d3.png)

I must say nbminer runs quite smoothly :)  input / critiques encouraged and needed :)

**Comment by Tom5051** [comment]
3080ti LHR should be getting 91mh/s all day long.  limit your power to 286watts and OC your memory to +1700 with MSI Afterburner

One of ours - | 6|3080ti| 91.47 M|74.2|  5277|     0|  0| 284|  46|  90|100|1455|10951| 100| 322.1 K|

try this with memory OC of +1700. leave gpu clock stock
nbminer -a ethash -o stratum+ssl://xxxx.xxxxx.xxxxx:5555 -u xxxxxxxxxx.Miner -pl 286 -lhr 74.2 -fan 100 -i 100 -lm 1 -lrv 0.1 -lrl 30 -lrt 360 -fee 1 -log -log-no-job -log-cycle 600

if it crashes, lower the memory OC by 50 and reboot, try again. It's a bit of an art to fine tune.
